### ISSUE TITLE
Birthday Null

### ISSUE THREAD

**Issue by Kantesh** [sromku/android-simple-facebook#112]
Even though i pass permission to get my profile birthday and friends birthday i am getting the field as null.

**Comment by eduardonatan** [comment]
Have you solved this? please, i m facing the same issue

**Comment by LiHongJian** [comment]
Yes, This is from facebook permission issue, in Facebook New version, you should submit request to get more permission such as birthday, Education / Working information etc....

**Comment by LiHongJian** [comment]
Yes, This was from facebook permission issue, in Facebook New version, you should submit request to get more permission such as birthday, Education / Working information etc....
